internal/distro/rhel9/qcow2.go

```diff
@@ -158,3 +143,31 @@ func openstackCommonPackageSet(t *imageType) rpmmd.PackageSet {
 
 	return ps
 }
+
+func qcowImageConfig(d distribution) *distro.ImageConfig {
+	ic := &distro.ImageConfig{
+		DefaultTarget: common.StringToPtr("multi-user.target"),
+	}
+	if d.isRHEL() {
+		ic.RHSMConfig = map[distro.RHSMSubscriptionStatus]*osbuild.RHSMStageOptions{
+			distro.RHSMConfigNoSubscription: {
+				DnfPlugins: &osbuild.RHSMStageOptionsDnfPlugins{
+					ProductID: &osbuild.RHSMStageOptionsDnfPlugin{
+						Enabled: false,
+					},
+					SubscriptionManager: &osbuild.RHSMStageOptionsDnfPlugin{
+						Enabled: false,
+					},
+				},
+			},
+		}
+
+	}
+	return ic
+}
+
+func mkQcow2ImgType(d distribution) imageType {
+	it := qcow2ImgType
```

Comment:
Yes, this came up here: https://github.com/osbuild/osbuild-composer/pull/3120#discussion_r1031490462
I wasn't going to change it if no other changes were necessary, but I'll do it now since it can lead to problems.